docs/blog/2025-01-03-december-release-notes.md

```diff
@@ -0,0 +1,48 @@
+slug: release-notes-december
+tags: [Release Notes]
+image: /img/blog/social-previews/rune.png
+authors:
+- name: Amani Albrecht
+  title: Chief of Staff at Rune  
+  url: https://www.linkedin.com/in/amanialbrecht
+  image_url: /img/blog/people/amani-albrecht.png
+  hide_table_of_contents: true
+---
+
+<head>
+  <title>New Updates! December 2025</title>
+  <meta property="og:title" content="New Updates! December 2025"/>
+</head>
+
+## 🛠️ App Improvements
+
+* Added quests into the app! Now gamers can earn gems by inviting friends, customizing avatars, playing games, and registering their email 💎🎮
+* Introduced daily rewards, allowing gamers to earn an increasing amount of gems by logging in each day 💎📅
+* Implemented gems for matching to foster a flourishing gem economy and make each matching session more meaningful 🤝
+* 📣 Showcased all the gems changes with a concise explainer announcement page.
+* Updated purchase flows for display names and avatars to match the gems for matching flow, ensuring consistency throughout the app!
+* The new login flow eliminated the need for an "update required" screen before login, streamlining access to the app 🚀
+* 🎯 Removed the "invite 3 friends" challenge banner and functionality as quests now replace this.
+* 🔴 Improved discoverability of unclaimed quests by marking them with a red dot, so you know you have gems waiting to be collected!
+* Released festive holiday avatar items for Christmas and New Years so everyone can get in the spirit 🎄🎉
+* Added a maintenance screen to manage downtime during major upgrades, keeping Rune gamers informed 📲
+
+## 🪲Bug Fixes
+
+* Disabled all the new gem sounds to avoid crashes while we fix them behind the scenes 🔇
+* ✨ Memoized each game’s comment list header component to prevent flickering!
+* Made sure quest headings are translated when the language is changed 🌐🔁
+* Fixed an issue where the Add Friend quest wasn’t opening the Invite Friend action sheet.
+* Updated our push code to only show one reward alert at a time, avoiding confusing spam 🚨
+* Resolved the problem where the profile page wasn't scrolling to show the remaining quests on Android phones 📱
+* Prevented errors by disabling multiple claims for gem rewards while the first tap is being processed.
+* Tweaked our email verification flow to respect the gamer's language settings on their phone 🌍
+* Made further improvements to our splash screen code to catch and address issues causing prolonged load times of 30s+.
+* Adjusted font sizing to accurately reflect the number of people who played games recently 🔠
+* 🛡️ Cleaned up various small errors in the new login flow to bulletproof the experience for everyone.
+* Busted a small but frustrating bug where occasionally icons were being swapped throughout the app!
+* Averted a crash on the daily rewards main screen by cleaning up some code 🥳
+* Upgraded React Native Reanimated, React Native Screens, and Facebook GIFs to remove occasional animation crashes on Android!
```

Comment:
I'm not crazy about publicizing the libraries that we use, maybe we could just say something like: 

```suggestion
* Upgraded some dependencies to remove occasional animation crashes on Android!
```